recipes/opencl-headers/all/conanfile.py

```diff
@@ -31,7 +31,15 @@ def package(self):
     def package_info(self):
         self.cpp_info.filenames["cmake_find_package"] = "OpenCLHeaders"
         self.cpp_info.filenames["cmake_find_package_multi"] = "OpenCLHeaders"
+        self.cpp_info.set_property("cmake_file_name", "OpenCLHeaders")
+
         self.cpp_info.names["cmake_find_package"] = "OpenCL"
         self.cpp_info.names["cmake_find_package_multi"] = "OpenCL"
+        self.cpp_info.set_property("cmake_target_name", "OpenCL::OpenCL")
+        self.cpp_info.set_property("pkg_config_name", "OpenCL")
+
         self.cpp_info.components["_opencl-headers"].names["cmake_find_package"] = "Headers"
         self.cpp_info.components["_opencl-headers"].names["cmake_find_package_multi"] = "Headers"
+        self.cpp_info.components["_opencl-headers"].set_property("cmake_target_name", "OpenCL::Headers")
+        self.cpp_info.components["_opencl-headers"].set_property("pkg_config_name", "OpenCL")
+
```

Comment:
```suggestion
```

You know you can accept suggestions through github? It saves time sometimes.

Reply:
yes, but I prefer to test that locally before committing